target/linux/ipq40xx/image/Makefile

```diff
@@ -257,6 +257,24 @@ define Device/engenius_ens620ext
 endef
 TARGET_DEVICES += engenius_ens620ext
 
+define Device/ezviz_cs-w3-wd1200g-eup
+	$(call Device/FitImage)
```

Comment:
Does the device support LZMA packed kernels? If so, an LZMA packed kernel saves quite some space on the SPI.

Reply:
Good question. I just tested this, and the shows "Unimplemented compression type 3" if I try to boot a LZMA image, so this does not seem to be supported.